src/worker/mod.rs

```diff
@@ -380,7 +406,7 @@ impl<
     where
         Q: AsRef<str>,
     {
-        use std::process;
+        self.forever = true;
```

Comment:
this is to be able to switch off the corresponding `select!` branch for normal (returning) `run` (as opposed to `run_to_completion`)